### PR TITLE
Cache ordering, stats, price, names

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: src/config/constants.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-export const BACKEND_URL = typeof Cypress !== 'undefined' ? 'http://localhost' : 'https://raendom-backend.z52da5wt.xyz';
+export const BACKEND_URL = typeof Cypress !== 'undefined' ? 'http://localhost' : 'http://localhost:3000';
```
Comment (mradkov) [inline src/config/constants.js]: backend url removed?

Comment (thepiwo) [reply src/config/constants.js]: my bad, sorry

**Comment by mradkov** [comment]
@thepiwo can we merge this?

**Comment by thepiwo** [comment]
@mradkov it depends on the backend being deployed, then we can merge (cc @kenodressel )

**Comment by thepiwo** [comment]
https://github.com/aeternity/tipping-community-backend/pull/74

**Pull request merged**